README.md

```diff
@@ -1,2 +1,18 @@
 # autolabcli
 Codebase for backward incompatible revision of autolabcli v0.1.1.
+
+[![codecov](https://codecov.io/gh/AutolabJS/autolabcli/branch/autolabcli-ng/graph/badge.svg)](https://codecov.io/gh/AutolabJS/autolabcli)
+[![Code Climate](https://codeclimate.com/github/AutolabJS/autolabcli/badges/gpa.svg)](https://codeclimate.com/github/AutolabJS/autolabcli)
+
+A client command line interface for submissions to [AutolabJS](https://github.com/AutolabJS/AutolabJS).
+
+## Commands ##
+* `autolabjs init -u <username> -p <password>` - Log into AutolabJS.
```

Comment:
`autolabjs init [-u <username> -p <password>]` - Log into AutolabJS.

The brackets are used by convention for optional part of the command. You can change the notation in a similar manner for all other commands.